src/graph_notebook/notebooks/04-Machine-Learning/Neptune-ML-06-Text-Encoding-Tutorial.ipynb

```diff
@@ -0,0 +1,1281 @@
+    }
+   ],
+   "source": [
+    "%%gremlin\n",
+    "g.V('job_26829').valueMap('Title')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "55b3bae1dc354cac8d3470ca7f33447e",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Tab(children=(Output(layout=Layout(max_height='600px', overflow='scroll', width='100%')), Output(layout=Layout…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%gremlin\n",
+    "g.E('app_5325').drop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Checking our data again, we see that two of five edges have now been removed. Only three edges are left."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "a43d985a5e3846f2a94c0e61aa5385d8",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Tab(children=(Output(layout=Layout(max_height='600px', overflow='scroll', width='100%')), Output(layout=Layout…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%gremlin\n",
+    "g.V('user_443931').out('apply').hasLabel('job').valueMap('Title')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 2: Export Data: Choose the Right Text Encoders for Text Properties\n",
+    "\n",
+    "**Note:** Before exporting data ensure that Neptune Export has been configured as described here: [Neptune Export Service](https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-data-export-service.html#machine-learning-data-export-service-run-export). \n",
+    "\n",
+    "The export process is triggered by calling to the [Neptune Export service endpoint](https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-data-export-service.html).  This call contains a configuration object, which specifies the link prediction task on the edge of `(\"user\", \"apply\", \"job\")` for this job recommendation problem.  The configuration also specifies the desired feature processing to apply.\n",
+    "\n",
+    "#### Configuring features\n",
+    "In machine learning, each property is transformed into a \"feature\", and these features are used by the model to make predictions.  When data is exported from Neptune, all properties of all vertices are included.  Each property is treated as a separate feature for the ML model.  Neptune ML does its best to infer the correct type of feature for a property. In many cases, the accuracy of the model can be improved by specifying information about the property used for a feature. \n",
```

Comment:
Should we say explicitly that exporting the data is for ML training purposes?

Reply:
fixed

Reply:
Discussed offline.

Instead of, "In machine learning, each property is transformed into a "feature", and these features are used to train a ML model and then to make predictions."

Suggested: "To make predictions on node or edge properties using Neptune ML, you can configure the properties as "features" and then export them into a format that is ready for data processing."